QueueManagerTest/DirectoryReaderTests.cs

```diff
@@ -1,26 +1,44 @@
 ﻿using System.Collections.Generic;
 using NUnit.Framework;
 using QueueManager;
+using System.IO;
+using System;
 
 namespace QueueManagerTest
 {
     [TestFixture]
-    public class NetflixReaderTests
+    public class DirectoryReaderTests
     {
         private DirectoryReader _directoryReader;
-        private string path;
+        private string upperLevelFolderName;
+        private string pathString;
+        private string appendedPathString;
+        private List<string> filesInUpperLevelFolder;
 
         [SetUp]
         public void SetUp()
         {
             _directoryReader = new DirectoryReader();
-            path = @"..\..\TestDir\";
+            upperLevelFolderName = @"..\..\TestDir\";
+            pathString = Path.GetFullPath(upperLevelFolderName);
+            Directory.CreateDirectory(pathString);
+            filesInUpperLevelFolder = new List<string> { "a1.iso", "a2.iso", "a3.iso", "a4.iso - Shortcut.lnk" };
+            foreach (string file in filesInUpperLevelFolder)
+            {
+                appendedPathString = Path.Combine(pathString, file);
+                File.Create(appendedPathString);
+            }
         }
 
         [Test]
         public void TestAddFilesInDirectoryToList()
```

Comment:
`$"{variableName} Some static text: {anotherVariableName}"` is an example of interpolated strings.

`someIEnumerable.Select(x => x)` is an example of a lambda expression in a Linq statement.

So they're two separate things. String interpolation came about as a favorable alternative to `string.Format("{0} this is a format string with two values in it {1}", firstValue, secondValue);`

String interpolation is better since it isn't prone to out of order parameters or an incorrect number of parameters.

A few Linq statements that I commonly use are Select, Where, FirstOrDefault, and OrderBy. Generally you can think of Linq as a concise way of replacing a foreach. If you have a complex foreach then it's probably better to be more expressive and use the foreach for readability.